resources/l10n/bg/LC_MESSAGES/loot.po

```diff
@@ -139,7 +139,7 @@ msgstr ""
 
 #: src/gui/state/game/game.cpp:512
 msgid "Cyclic interaction detected between \"%1%\" and \"%2%\": %3%"
-msgstr "Засечено е циклично взаимодействие между „%1%“ и „%2%“: %3%"
+msgstr "Засечено е циклично взаимодействие между „%1%“ и „%2%“: %3%“"
```

Comment:
That last closing double quote doesn't have an opening quote.

Reply:
Oops...